parts/kubernetesmastercustomscript.sh

```diff
@@ -249,6 +249,23 @@ function ensureEtcd() {
     done
 }
 
+function ensureEtcdDataDir() {
+    etcdStarted=1
+    mount | grep /dev/sdc1 | grep /var/lib/etcddisk
+    if [ "$?" = "0" ]
+    then
```

Comment:
nit: I assume you copy/pasted one that was looping? Since there's no loop here, there's no need for the second if. You can just early return if the exit code was zero and "exit 1" if you make it out of the `if`. (Or inversed)